Makefile

```
@@ -0,0 +1,6 @@
+LDFLAGS=-w -extldflags
+
+build: ## Runs go build on repo
+	# Run go build and generate scorecard executable
+	CGO_ENABLED=0 go build -o scorecard-action -trimpath -a -tags netgo -ldflags '$(LDFLAGS)'
+	
```

```suggestion
	CGO_ENABLED=0 go build -o scorecard-action -trimpath -a -tags netgo -ldflags '$(LDFLAGS)'

```

I think the newline has an errant tab here.